spec/models/activity_spec.rb

```diff
@@ -32,7 +32,7 @@
     let(:expected_attributes) do
       {
         id: nomis_scheduled_event['eventId'],
-        start_time: Time.parse(nomis_scheduled_event['startTime']),
+        start_time: Time.parse(nomis_scheduled_event['startTime']), # rubocop:disable Rails/TimeZone
```

Comment:
From the comment on the activity class:
Nomis startTime does not include a zone so we shouldn't use our local zone with Time.zone.parse and arbitraily add a timezone